src/components/insights/OverallComparer.ts

```diff
@@ -42,7 +42,7 @@ export function overallComparer(
 			}, 1000);
 			return;
 		}
-		alertDiv.textContent = comparer(arrayConverted, focusedData);
+		alertDiv.textContent = messagecreator(arrayConverted, focusedData);
```

Comment:
Is this correct? Shouldn't it be `messageCreator` (with a capital C)

Reply:
This logic also changes in a following PR :)